### PR TITLE
Improve segment form grid layout [MAILPOET-4039]

### PR DESCRIPTION
**Pull request by NeosinneR**
[MAILPOET-4039]

[MAILPOET-4039]: https://mailpoet.atlassian.net/browse/MAILPOET-4039?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Testing instructions are inside the Jira.

### CODE REVIEW

File: assets/css/src/components-plugin/_segments.scss [hunk 1]
```diff
@@ -5,7 +5,8 @@
   grid-column: span 2;
 }
 
-.mailpoet-segments-segments-section .mailpoet-grid-three-columns {
+.mailpoet-segments-grid {
+  grid-template-columns: 200px 1fr 1fr;
```
Comment (costasovo) [inline assets/css/src/components-plugin/_segments.scss:9]: I noticed that this rule overrides the rule for mobile view defined for `mailpoet-grid-three-columns` and it results in worse UX on mobiles. There probably won't be many users who configure segments on mobile, but I guess it should be an easy fix.
Screenshot 2021-12-29 at 11 04 55

**Review by costasovo — CHANGES_REQUESTED** [review]
Good job! I found one minor issue that probably would be nice to fix.

File: assets/css/src/components-plugin/_segments.scss (hunk 1 above)
Comment (NeosinneR) [reply assets/css/src/components-plugin/_segments.scss:9]: Good catch! Fixed

**Review by costasovo — APPROVED** [review]
👍

**Comment by rohankacharegs** [comment]
GlobalStep Verified

**Pull request merged**